### DISCUSSION TITLE
请问在什么情况下用户目录的 owner 和 group 会被改为 1panel (1000) 呢？

### DISCUSSION THREAD

**Discussion by genment** [1Panel-dev/1Panel#5815] [Q&A]
### 1Panel 版本

v1.10.11-lts

### 问题描述

最近ssh密钥登录总是失败，要用密码登录才行，但也没太在意。然后今天就发现 /root 用户目录的 owner 和 group 都变成了 1panel (1000) 。改回 root (0) 之后 ssh 密钥就可以正常登录了。

我100%肯定没有手动更改过，所以我想知道在使用 1panel 的过程中，有没有哪个操作可能会更改了用户和组？

### 重现步骤

不知道

### 期待的正确结果

不会更改用户和用户组，或者发出提醒？

**Comment by zhengkunwang223** [comment]
您好 当前修改 1000:1000 的操作有
网站目录 和 上传文件 
都需要主动操作，而且并不会去操作 /root 目录 ，考虑是不是执行 chown 的时候误操作了？
